docs/physics/montecarlo/initialization.ipynb

```diff
@@ -67,7 +67,7 @@
     "\n",
```

Comment:
<p>The difference <code>r_boundary_inner</code> and<code> temperature_inner.value</code>	means that the temperature property of packet_source needs to handle astropy quantities correctly, so that the .value is not required.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>